cli/tests/pcluster/validators/test_all_validators.py

```diff
@@ -406,6 +412,7 @@ def test_scheduler_plugin_all_validators_are_called(test_datadir, mocker):
                 "CapacityReservationResourceGroupValidator",
                 "DatabaseUriValidator",
                 "InstanceTypePlacementGroupValidator",
+                "FsxDescribeVolumesValidator",
```

Comment:
We do not want to add new validators in the production code

Reply:
I am also concerned about this even though this exact line actually skips the check of the new validator. It seems that this approach to truly be complete could introduce risks, but the alternative approach e.g. from `test_shared_storage.py` is unclear. So, I chose this way of adding a validator, but not having it enforced as a potential middle ground between the riskier and the convoluted approaches.

Reply:
Let's remove the validator: we want to test an existing behaviour leveraging mocking, not introducing new validation logic into the production code.

Reply:
Ok